lua/nordic/utils.lua

```diff
@@ -46,82 +59,7 @@ function M.hex_to_rgb(str)
 end
 
 function M.rgb_to_hex(r, g, b)
-    return '#' .. string.format('%x', r) .. string.format('%x', g) .. string.format('%x', b)
-end
-
-function M.rgb_to_hsv(r, g, b)
-    r, g, b = r / 255, g / 255, b / 255
-    local max, min = math.max(r, g, b), math.min(r, g, b)
-
-    local h, s, v
-    v = max
-
-    local d = max - min
-    if max == 0 then
-        s = 0
-    else
-        s = d / max
-    end
-
-    if max == min then
-        h = 0
-    else
-        if max == r then
-            h = (g - b) / d
-            if g < b then h = h + 6 end
-        elseif max == g then
-            h = (b - r) / d + 2
-        elseif max == b then
-            h = (r - g) / d + 4
-        end
-        h = h / 6
-    end
-
-    return h, s, v
-end
-
-function M.hsv_to_rbg(h, s, v)
-    local r, g, b
-
-    local i = math.floor(h * 6)
-    local f = h * 6 - i
-    local p = v * (1 - s)
-    local q = v * (1 - f * s)
-    local t = v * (1 - (1 - f) * s)
-
-    i = i % 6
-
-    if i == 0 then
-        r, g, b = v, t, p
-    elseif i == 1 then
-        r, g, b = q, v, p
-    elseif i == 2 then
-        r, g, b = p, v, t
-    elseif i == 3 then
-        r, g, b = p, q, v
-    elseif i == 4 then
-        r, g, b = t, p, v
-    elseif i == 5 then
-        r, g, b = v, p, q
-    end
-
-    return r * 255, g * 255, b * 255
-end
-
-function M.darken(hex, amount)
-    local r, g, b = M.hex_to_rgb(hex)
-    local h, s, v = M.rgb_to_hsv(r, g, b)
-    v = v * ((1 - amount) / 1)
-    r, g, b = M.hsv_to_rbg(h, s, v)
-    return M.rgb_to_hex(r, g, b)
-end
-
-function M.lighten(hex, amount)
```

Comment:
As far as I can tell, none of these are being used anywhere...

If we want to darken, can't we just blend `#000000` or `#FFFFFF` if we want to lighten?

This seems like a lot of code to be keeping around if it's not doing anything. What do you think? @AlexvZyl

Reply:
Also, I am not entirely convinced that `lighten` even works...

Reply:
Yeah I initially just added those while I was working on the theme.  Never needed them.  Good thing you are removing them, we can always just add them again.

Reply:
Just blending with black or white does not darken/lighten as far as I am aware.  Pretty sure you want to use the HSV format for that.  But let's not worry about it.